docs/02-Deploying/03-Configuration.md

```diff
@@ -2043,6 +2043,10 @@ It is strongly recommended that at least one admin user is set up to use the tra
 based log in so that there is a fallback method for logging into Fleet in the event of SSO
 configuration problems.
 
+### Enabling SSO for existing users in Fleet
+
+To enable SSO for existing users as an admin, go to the Settings page, then click on the Users tab. Locate the user you want to enable SSO for and on the actions dropdown menu for that user, click on "Enable single sign on".
```

Comment:
> To enable SSO for existing users as an admin

As discussed, It may seem obvious, but I was confused about whether this meant that SSO could be enabled for all existing users or only for admin users. Please rephrase to make it clear that admins can enable SSO for existing users.

Reply:
Let me know what you think